### PR TITLE
Literature input in PIS

### PR DESCRIPTION
**Pull request by mbdebian**
This pull request resolves opentargets/platform#1922

### CODE REVIEW

**Comment by mbdebian** [comment]
I just realized that I should probably update the documentation as well, although that doesn't really need a PR

File: config.yaml [hunk 1]
```diff
@@ -373,3 +373,8 @@ target:
       path: target-inputs/ensembl
       jq: '.genes[] | {id: .id, biotype: .biotype, description: .description, end: .end, start: .start, strand: .strand, chromosome: .seq_region_name, approvedSymbol: .name, transcripts: .transcripts, signalP: .SignalP, uniprot_trembl: ."Uniprot/SPTREMBL", uniprot_swissprot: ."Uniprot/SWISSPROT"}'
   miniset: yes
+literature:
```
Comment (JarrodBaker) [inline config.yaml:376]: We also need the raw files provided by EPMC from `gs://otar025-epmc`

**Review by JarrodBaker — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (mbdebian) [reply config.yaml:376]: I mean at the ETL stage, we don't copy that information into our pre-data-releases bucket

**Review by JarrodBaker — APPROVED** [review]
Let's carry on doing what we're doing regarding the ETL reading from the ePMC bucket. If we start running into consistency issues we'll revisit the issue then.

**Pull request merged**